### ISSUE TITLE
[FEATURE] Lighten or Desaturate Color Variables in Dark Mode

### ISSUE THREAD

**Issue by glavrjk** [ColorlibHQ/AdminLTE#3230]
**Is your feature request related to a problem? Please describe.**
Regards. I have a small problem with the new dark mode of the 3.1-RC, although for me the choice of dark tones is correct. The default color palette is too saturated and in my project where the primary and secondary colors are quite strong it really looks awful.
**Describe the solution you'd like**
My question is if there is a way to overwrite the variable values ​​when dark mode in the body class is active? Something like this:
```scss
if ($dark_mode == true) {
   $primary = lighten($primary, $percent);
}

// OR

.dark-mode {
  $primary = lighten($primary, $percent);
}

```
Thank you and sorry for my google translator english... 😄

**Comment by danny007in** [comment]
I will do this implementation

**Comment by glavrjk** [comment]
> I will do this implementation

Regards. Thanks for your hard work. I have verified your implementation to my issue, but I think it will not solve my problem, although it helps a lot and I appreciate the effort.

The point is that in order to implement the dark colors you would have to declare the dark css from the <head>. That is, statically implement the dark palette to make it work. Right?

What if through the backend I have configured the users to select the theme they want, whether it is light or dark. I could parameterize the css of the head according to the value of user.dark_mode but I use webpack encore for the handling of dependencies and I cannot or do not know how I would tell it to compile adminlte.css or adminlte-dark.css

Any suggestion? Thank you...

**Comment by danny007in** [comment]
Check https://stackoverflow.com/questions/14292997/changing-style-sheet-javascript

**Comment by danny007in** [comment]
> 
> 
> > I will do this implementation
> 
> Regards. Thanks for your hard work. I have verified your implementation to my issue, but I think it will not solve my problem, although it helps a lot and I appreciate the effort.
> 
> The point is that in order to implement the dark colors you would have to declare the dark css from the head. That is, statically implement the dark palette to make it work. Right?
> 
> What if through the backend I have configured the users to select the theme they want, whether it is light or dark. I could parameterize the css of the head according to the value of ( user.dark_mode ? true : false ) but I use webpack encore for the handling of dependencies and I cannot or do not know how I would tell it to compile adminlte.css or adminlte-dark.css
> 
> Any suggestion? Thank you...

Check https://stackoverflow.com/questions/14292997/changing-style-sheet-javascript

**Comment by REJack** [comment]
I will tackle this feature in v4, caused that I want to release v3.1.0 soon.

**Comment by danny007in** [comment]
i will try this in 3.1 for button only for now

**Comment by danny007in** [comment]
Fixed in https://github.com/ColorlibHQ/AdminLTE/pull/3492

**Comment by glavrjk** [comment]
> Fixed in #3492

Thank you so much. Waiting for release 3.1
